sdf-schema.json

```diff
@@ -111,7 +112,8 @@
           "properties": {
             "$ref": {
               "type": "string"
-            }            
+            },
+          "additionalProperties": false        
```

Comment:
Reading through this <https://json-schema.org/understanding-json-schema/reference/object.html>, it looks like this can be used to also effect Weave's "extendable" qualifier.